src/Illuminate/Console/Scheduling/Schedule.php

```diff
@@ -77,6 +83,13 @@ class Schedule
      */
     protected $mutexCache = [];
 
+    protected ScheduleAttributes $attributes;
+
+    /**
+     * @var array<int, ScheduleAttributes>
```

Comment:
Missing a doc block description here (correct the suggested message if it's not suitable):

```suggestion
     * The nested commands to schedule.
     *
     * @var array<int, ScheduleAttributes>
```